modules/openapi-generator/src/main/resources/JavaSpring/libraries/spring-boot/pom.mustache

```diff
@@ -24,7 +27,7 @@
     <parent>
         <groupId>org.springframework.boot</groupId>
         <artifactId>spring-boot-starter-parent</artifactId>
-        <version>{{#java8}}2.3.3.RELEASE{{/java8}}{{^java8}}1.5.12.RELEASE{{/java8}}</version>
+        <version>2.5.8</version>
```

Comment:
Intentionally changing this? Seems to be java8 related as opposed to OAS3

Reply:
The generated pom include outdated, unsupported dependencies. spring-boot 3.3 is EOL. I upgraded several outdated dependecies "en passant".